README.md

```diff
@@ -1,2 +1,72 @@
-# tacoctl
+# tacoctl 
 CLI tool for TACOPLAY
+
+------
+
+## What is the tacoctl
+
+The tacoctl is next deploy management tool of [tacoplay](github.com/openinfradev/tacoctl). It support a Cluster
```

Comment:
The tacoctl project is a CLI tool for declarative management of kubernetes and underlying infrastructure, mainly leveraging the Kubernetes Cluster API. It builds and manage a Kubernetes Cluster either on Baremetal or OpenStack VMs. 

This project is the ongoing effort to produce next generation of existing [tacoplay](github.com/openinfradev/tacoplay). Currently, the tacoctl is at an very early development stage.